### DISCUSSION TITLE
File Extension Issue

### DISCUSSION THREAD

**Discussion by MaximusGS13** [eltos/PasteIntoFile#51] [Q&A]
I recently upgraded to 5.3.1.0.  Since then, the file extension defaults to .emf regardless of the file type i save it to last, which is almost always .jpg.  Can someone tell me how to get it to default to .jpg instead?

**Comment by eltos** [comment]
Similar issue as #45 

Can you check [Version 5.3.2](https://github.com/eltos/PasteIntoFile/releases/tag/v5.3.2) please?
